src/explore-education-statistics-frontend/.env

```diff
@@ -3,3 +3,4 @@ DATA_API_BASE_URL=http://localhost:5000/api
 NOTIFICATION_API_BASE_URL=http://localhost:7073/api
 GA_TRACKING_ID=
 PUBLIC_URL=http://localhost:3000
+PERMITTED_EMBED_URL_DOMAINS=https://department-for-education.shinyapps.io,https://dfe-analytical-services.github.io
```

Comment:
Not really keen on having comma-separated values here. It seems unnecessary since the `.env` will be source controlled and exposed. Also makes the syntax a little bit gnarly when you're mapping over these values in `server.js`. It would be simpler to just define an array of values in something such as `config.ts`

```typescript 
export const allowedEmbedDomains = [
    'https://department-for-education.shinyapps.io',
    'https://dfe-analytical-services.github.io',
  ];
```


```
 frameSrc: ["'self'", ...allowedEmbedDomains],
```